import.go

```diff
@@ -91,6 +91,13 @@ func importBuilderFromImage(store storage.Store, options ImportFromImageOptions)
 	}
 	img, err := is.Transport.GetStoreImage(store, ref)
 	if err != nil {
+		img2, err2 := store.Image(options.Image)
```

Comment:
Ok.  I was thinking you could have thrown the err from inside of the block preventing the second check.
```
if err2 == nil {
		img = img2
		err = nil
} else {
  		return nil, errors.Wrapf(err, "unable to locate image") 
}
```